index.d.ts

```diff
@@ -24,6 +24,7 @@ export interface FastifyOAuth2Options {
   credentials: Credentials;
   callbackUri: string;
   callbackUriParams?: Object;
+  getTokenParams?: Object;
```

Comment:
can you add a test for this new type? It seems it's a function, not an object.

Reply:
When calling `getAccessTokenFromAuthorizationCodeFlow` those parameters are added to the payload of the `o.oauth2.getToken` POST request. I know the name `getTokenParams` is not optimal, but it is an object. 

For example if it's set to:

```
getTokenParams: {
  client_id: 'id',
  client_secret: 'secret'
}
``` 

and setting `export DEBUG=*simple-oauth2*`, then you can see that this parameters are added to the payload when retrieving the OAuth tokens:

```
 simple-oauth2:client Creating request to: (POST) /oauth/token +0ms
 simple-oauth2:client Using request options: {"headers":{"Authorization":"Basic XXX","Content-Type":"application/x-www-form-urlencoded"},"payload":"grant_type=authorization_code&client_id=id&client_secret=secret&code=3dd...71&redirect_uri=https%3A%2F%2F..."} +0ms
```

Can you think about a better naming? Maybe `tokenRequestParams?: Object`?

I'll have a look into testing in the meantime.